lib/samson/secrets/db_backend.rb

```diff
@@ -21,6 +21,19 @@ def read(id)
           secret_to_hash secret
         end
 
+        # Not implemented, just bogus values to be able to debug UI in development+test
+        def history(_id)
```

Comment:
shouldn't this be stubbed in test? looks strange if this secret backend otherwise works to add dummy data here.

Reply:
I'm using this for development / styling, where I don't have stubs